raiden/utils/crypto.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
 
-import bitcoin
+from binascii import unhexlify
+
+from coincurve import PrivateKey
 
 
 def privtopub(raw_privkey):
-    pubkey = bitcoin.privtopub(raw_privkey)
-    raw_pubkey = bitcoin.encode_pubkey(pubkey, 'bin_electrum')
-    assert len(raw_pubkey) == 64
-    return raw_pubkey
+    # either bytes or hex
+    if not isinstance(raw_privkey, bytes):
```

Comment:
Do we need this check? At least [here](https://github.com/raiden-network/raiden/blob/master/raiden/tests/utils/blockchain.py#L249) the argument type is `bytes`.

Reply:
If [here](https://github.com/raiden-network/raiden/blob/8c49717634e8e4303b10275c906f5fd446c374d4/raiden/accounts.py#L209) is in bytes, then most likely not.

Reply:
I checked and `decode_keystore_json` returns bytes. So we can safely remove the type check and conversion.

Reply:
I'd also say we can move this function to the `__init__` file.